result_collector/aggregate_results.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import re
+
+def get_number(filename):
+    return re.findall(r'\d+', filename)[0]
+
+def load_previous_cumulative_arrays():
+    cum_loss = np.load("/input/cumloss.npy")
+    cum_indeces = np.load("/input/cumindeces.npy")
+    return cum_loss, cum_indeces
+
+def load_previous_results(tag):
```

Comment:
Do you delete the files that are already processed?

Reply:
It is good idea. But unfortunately docker doesn't have access to eos directly

Reply:
It might then make sense to have a script we run manually on lxplus to clean up every now and again.

Also, how do you keep track of what has been processed?